lib/discovery.js

```diff
@@ -131,39 +131,7 @@ async function onDiscovery(peripheral) {
     mqtt.send(dev.advertise_topic, JSON.stringify(mqttData));
   }
 
-  peripheral.advertisement.serviceData.forEach(function (d) {
```

Comment:
Hi - could be wrong here, but doesn't this remove a bunch of other MQTT output that could just have easily been turned off by config options?

This will break EspruinoHub for a whole bunch of people

Reply:
Thanks! I hope not. I'll check again. Perhaps I missed something when merging the changes

Reply:
The changes just look like this because of an incorrect merge. I made this code a long time ago ) . a similar code is sent to mqtt below
Please test it.

Reply:
the duplication was leaked with this commit. I apologize https://github.com/espruino/EspruinoHub/commit/a0366f2b5e4602a22e60d4d03e1d87598d04baae#diff-7a037e43d9e32404e836f8529f966389fde35e986553f87f3a6967918f7debfdR134

Reply:
No problem - any chance you could fix that bit, and then I think we're good to merge? Thanks!

Reply:
I think yes. if there are regressions, mention me. I will try to help if possible.